src/MessagePack.UniversalCodeGenerator/Properties/launchSettings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "MessagePack.UniversalCodeGenerator": {
+      "commandName": "Project",
+      "commandLineArgs": "-i .\\sandbox\\shareddata\\shareddata.csproj -o .\\sandbox\\Sandbox\\Generated.cs",
+      "workingDirectory": "D:\\git\\MessagePack-CSharp"
```

Comment:
Is environment depenent config place to repos okay?

Reply:
No, sorry. I accidentally checked this file in.